dev/bots/test.dart

```diff
@@ -30,7 +29,7 @@ const String arrow = '⏩';
 const String clock = '🕐';
 
 const Map<String, ShardRunner> _kShards = <String, ShardRunner>{
-  'analyze': _analyzeRepo,
+  // 'analyze': _analyzeRepo, handled by analyze.dart
```

Comment:
Just remove this line, and the ones below.  You might add a comment above the declaration just saying that this only covers shards handled by this script.